pkg/provisioner/ironic/raid_test.go

```diff
@@ -12,6 +12,7 @@ import (
 func TestBuildTargetRAIDCfg(t *testing.T) {
 	var TRUE = true
 	var FALSE = false
+	physicalDisks := make([]interface{}, 0)
```

Comment:
I think it would be nice to have at least one case where the hardware physical volumes were set (and possibly some others to verify the error conditions)

Reply:
Oh alright. I'll add one right away.